sysutils/tiptop/Portfile

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github  1.0
+PortGroup           python  1.0
+
+github.setup        nschloe tiptop 0.0.16 v
+github.tarball_from archive
+revision            0
+
+description         \
+    Command-line system monitoring
+
+long_description    \
+    tiptop is a command-line system monitoring tool in the spirit of top. It \
+    displays various interesting system stats, graphs it, and works on Linux \
+    and macOS.
+
+platforms           darwin
+license             MIT
+categories          sysutils
+maintainers         {gmail.com:herby.gillot @herbygillot} openmaintainer
+
+checksums           rmd160  a4ae261c51af8ebb5b736c6c2cee39a8658837ad \
+                    sha256  73f8a4367f05acc90a118554bfd5bfd9b645a30b1de4985740b1cf6a1a6486c2 \
+                    size    14932
+
+python.default_version      39
+python.pep517               yes
+
+depends_lib-append  port:py${python.version}-setuptools
+
+depends_run-append  port:py${python.version}-cpuinfo \
+                    port:py${python.version}-distro \
+                    port:py${python.version}-importlib-metadata \
+                    port:py${python.version}-psutil \
+                    port:py${python.version}-textual
+
+test.run            yes
+test.env-append     PYTHONPATH=./build/lib
+test.cmd            ${python.bin}
+test.pre_args       -c \"import tiptop\; tiptop.run()\"
+test.args           --version
```

Comment:
it looks like there are actual tests in the GitHub repository, can these be run instead of just importing it and returning a version?

Reply:
Done.